pkgs/development/libraries/aspell/dictionaries.nix

```diff
@@ -1,137 +1,217 @@
-{stdenv, fetchurl, aspell, which}:
+{lib, stdenv, fetchurl, aspell, which}:
+
+with lib;
+
+/* HOWTO:
```

Comment:
This only considers the method where dicts are installed separate from aspell, and not `aspellWithDicts`. Perhaps there is a better place for this comment? Otherwise, it's good to have the explanation!